django/test/runner.py

```diff
@@ -571,18 +571,25 @@ def run(self, result):
 
         test_results = pool.imap_unordered(self.run_subsuite.__func__, args)
 
+        timeout = 64  # ms
```

Comment:
I experiemented with another approach where I wrapped the body of `_init_worker` in a `try/except`, and set a flag (`init_failed`) in the except, and accessed it in the workers via a lock (shared state), but the bloating of the diff told me I was going in the wrong direction (?)